objectbox/test/boxnn_test.dart

```diff
@@ -0,0 +1,88 @@
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:test/test.dart';
+import 'package:objectbox/objectbox.dart';
+import 'entity.dart';
+import 'entity2.dart';
+import 'test_env.dart';
+
+// Tests non-nullable fields
```

Comment:
Maybe add a note that this is basically a copy of the regular box tests?

Reply:
but it is not really, it's much simplified here. added the comment to datetime field test which is pretty much the same